e107_plugins/forum/templates/forum_post_template.php

```diff
@@ -13,21 +13,35 @@
 // New in v2.x - requires a bootstrap theme be loaded.  
 
 //$FORUM_POST_TEMPLATE['caption']		= "Custom caption";
+
+$SC_WRAPPER['FORUM_POST_AUTHOR'] = "<div class='form-group'>
+										<label for='name' class='col-sm-3 control-label'>".LAN_FORUM_3010."</label>
+										 <div class='col-sm-9'>{---}</div>
+									</div>
+";
+$SC_WRAPPER['FORUM_POST_SUBJECT'] = "<div class='form-group'>
+										<label for='name' class='col-sm-3 control-label'>".LAN_FORUM_3011."</label>
+										 <div class='col-sm-9'>{---}</div>
+									</div>
+";
+
+// Replaces legacy? {FORUM_POST_OPTIONS_LABEL} shortcode...
+$SC_WRAPPER['FORUM_POST_OPTIONS'] = "<div class='form-group'>
+										<label for='name' class='col-sm-3 control-label'>".LAN_FORUM_8013."</label>
+										 <div class='col-sm-9'>{---}</div>
+									</div>
+";
+
+	
```

Comment:
I think you may be missing my point. If you have removed HTML from the shortcode and put it in a wrapper, and a v1.x theme is using a custom template, it will be missing that HTML unless they manually edit their theme and add those wrappers. This is my concern.